provisioning/miniprov/hare_mp/utils.py

```diff
@@ -185,7 +188,9 @@ def save_drives_info(self):
     def get_drive_info_from_consul(self, path: Text, machine_id: str) -> Disk:
         hostname = self.get_hostname(machine_id)
         disk_path = json.loads(str(path)).lstrip(os.sep)
-        drive_data = self.kv.kv_get(f'{hostname}/{disk_path}')
+        drive_data = None
+        while (not drive_data or drive_data is None):
+            drive_data = self.kv.kv_get(f'{hostname}/{disk_path}')
```

Comment:
I was wondering, what if some node's prepare stage is failed for some reason?
I think we should not worry about it as anyway the exception will be there for that node and deployment will fail, right?